source/form.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="ru" class="page">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Седона</title>
+  </head>
+  <body>
+    <header class="page-header">
+      <a class="page-header__logo">
+        <img src="img/logo.svg" width="102" height="84" alt="Логотип Седона">
+      </a>
+      <nav class="main-nav">
+        <button class="main-nav__open-menu"><span class="visually-hidden">Открыть меню</span></button>
+        <div class="main-nav_wrapper">
```

Comment:
```suggestion
        <div class="main-nav__wrapper">
```
На остальных страницах тоже исправь.

Reply:
у меня все корректо исправлено, не совсем понял ошибку.
На всех страницах так и указано:  <div class="main-nav__wrapper">

по поводу скрытых блоков - они на странице формы, т.к. ты сам сказал, что проверяем сначала первое задание, а потом соответственно делаю бонусное, бонусное я еще не делал)

Reply:
Точно!